src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item-info/ProteinsListItemInfo.tsx

```diff
@@ -93,6 +89,10 @@ const ProteinsListItemInfo = (props: Props) => {
   const { product } =
     transcriptWithProteinDomains?.product_generating_contexts[0] || {};
 
+  const uniprotXref = product?.external_references.find(
+    (xref) => xref.source.id == 'Uniprot/SWISSPROT'
```

Comment:
We have already got a constant for this string in `ProteinListItem`: https://github.com/Ensembl/ensembl-client/blob/ebe7c8ac5712a2a55dc22895638b67f7237b2e0a/src/ensembl/src/content/app/entity-viewer/gene-view/components/proteins-list/proteins-list-item/ProteinsListItem.tsx#L37.

We'll probably need to export it from that file to make it reusable or add it to some helper file.